tests/test_playbooks/scc_account.yml

```diff
@@ -0,0 +1,83 @@
+---
+- hosts: localhost
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+  - include: tasks/organization.yml
+    vars:
+      organization_state: present
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_name: 'Test'
+      organization_name: 'Test Organization'
+      scc_account_state: absent
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_name: 'Test2'
+      organization_name: 'Test Organization'
+      scc_account_state: absent
+- hosts: tests
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_password: 'testpass'
+      scc_account_login: 'testuser'
+      scc_account_base_url: 'https://scc.suse.com'
+      test_connection: false
+      scc_account_state: present
+      expected_change: true
+
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_state: present
+      expected_change: false
+
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_name: 'Test'
+      scc_account_updated_name: 'Test2'
+      scc_account_interval: 'monthly'
+      test_connection: false
+      scc_account_state: present
+      expected_change: true
+
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_name: 'Test2'
+      scc_account_interval: 'monthly'
+      scc_account_state: present
+      expected_change: false
+
+  - include: tasks/scc_account.yml
+    vars:
+      scc_account_name: 'Test2'
+      test_connection: false
+      scc_account_state: synced
+      expected_change: true
```

Comment:
This should be expected to fail, because we do not provide proper credentials.
The real problem can however be in the way we report the task in foreman_scc_manager.
I think, either we fix that one now, or we skip that test, until then.